### ISSUE TITLE
matrix inverse get wrong results

### ISSUE THREAD

**Issue by xianlin** [koggdal/matrixmath#1]
Example:

``` javascript
var matrix = new Matrix(2, 2).setData(
  0, -600,
  240, 240
);
matrix.invert();
console.log("test: ", matrix.toLogString());

inverted Matrix A:  [
  0.0016666666666666668  -0.004166666666666667
  -0.0016666666666666668  0
]
```

The above Matrix is **wrong**. here are the correct Answer:

```
  1/600       1/240     
  -1/600      0  

```

May I know why the result is wrong? your 3x3 matrix inverse seems OK.

```
var matrix = new Matrix(3, 3).setData(
  3, 0, 2,
  2, 0, -2,
  0, 1, 1
);
matrix.invert();

> console.log(matrix.toLogString());
[
  0.2  0.2  0
  -0.2  0.3  1
  0.2  -0.3  0
]
```

**Comment by koggdal** [comment]
In what way is that wrong? If you compute your divisions, you'll get the numbers in the result matrix. This is JavaScript and it doesn't represent numbers by divisions.

**Comment by xianlin** [comment]
The 1/240 should be positive, but the result `-0.004166666666666667` is negative.

**Comment by koggdal** [comment]
Oh! Missed that detail, sorry. I'll have a look later today. Thanks!

**Comment by koggdal** [comment]
It's now fixed and published on npm as 2.2.1. Thanks for letting me know! :)

https://github.com/koggdal/matrixmath/releases/tag/v2.2.1
